custom_components/reflex_clerk_api/clerk_provider.py

```
@@ -80,13 +85,69 @@ def register_dependent_handler(cls, handler: EventCallback) -> None:
-        """Sets the max time to wait for initial auth check before running other on_load events.
+        """Set the frontend auth fallback timeout value.
 
-        Note: on_load events will still be run after a timed out auth check.
-        Check ClerkState.auth_checked to see if auth check is complete.
+        Auth-gated on_load events are normally flushed by the frontend Clerk sync
+        event. If Clerk never reports loaded or token retrieval stalls, the
+        frontend releases queued on_load events after this timeout without
+        changing the current auth state.
         """
+        if seconds < 0:
+            raise ValueError("auth wait timeout must be non-negative")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)